### PR TITLE
Exercise02

### PR DESCRIPTION
**Pull request by IdanAsh14**
A simple application that has 2 main buttons (image buttons), one labeled “Stark” and one labeled “Lannister”. Upon tapping a button, a new activity will be launched and display the proper character names and images in a RecyclerView control.

### CODE REVIEW

File: Exercise02/app/src/main/java/com/example/GameOfThrones/Data.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.example.GameOfThrones;
+
+public class Data {
```
Comment (ghost) [inline Exercise02/app/src/main/java/com/example/GameOfThrones/Data.java:3]: Try to think about a more intuitive name for your class.
Data is too general, is it a Character maybe?

File: Exercise02/app/src/main/java/com/example/GameOfThrones/Data.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    public int imageId;
+    public String txt;
```
Comment (ghost) [inline Exercise02/app/src/main/java/com/example/GameOfThrones/Data.java:5]: 1. In general, it is not recommended to "shorten" keywords, why txt and not text?
2. Same as the comment above, try to think about less general property name, what text is it? I can't refer to the text this class is about to hold from this general name.

**Review by ghost — APPROVED** [review]
In general it looks good, everything runs as expected.

**Comment by IdanAsh14** [comment]
I got the comments, implement them in the following apps.

**Pull request merged**